ucsm_apis/admin/user.py

```diff
@@ -211,7 +221,7 @@ def user_role_add(handle, user_name, name, descr=None, **kwargs):
     Args:
         handle (UcsHandle)
         user_name (string): username
-        name (string): rolename
+        name (string): rolename, also allowed comma separated roles
```

Comment:
rolename, also allowed comma separated roles -> rolename or a comma separated string of roles